recipes/ntp/ntp.inc

```diff
@@ -30,6 +30,8 @@ EXTRA_OECONF_CRYPTO = "--without-crypto"
 EXTRA_OECONF_CRYPTO:USE_crypto = "--with-crypto"
 DEPENDS:>USE_crypto = " libcrypto"
 
+EXTRA_OECONF += "--disable-debugging"
```

Comment:
OK, but since I currently happen to be doing ntp debugging on a project, can I selfishly ask you to add a USE flag to enable it?